hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/keyvalue/TestTarContainerPacker.java

```diff
@@ -382,7 +382,7 @@ private File writeDbFile(
 
   private File writeSingleFile(Path parentPath, String fileName,
       String content) throws IOException {
-    Path path = parentPath.resolve(fileName);
+    Path path = parentPath.resolve(fileName).normalize();
```

Comment:
Change looks OK and the test now passes. Do you know why `normalize` is needed in Java 17 but not in earlier versions? Has the behavior of `resolve` changed slightly?